### DISCUSSION TITLE
Running multiple instances of the same program with the help of firejail

### DISCUSSION THREAD

**Discussion by haxhash** [netblue30/firejail#4072] [Q&A] [answered]
I am interested in a survey of how I can run one and the same program many times with the help of FireJail, as separate sessions. 
When I try to do it with stimulus, I get this:
```
Steam client's requirements are satisfied
/home/alex/.steam/debian-installation/ubuntu12_32/steam -nominidumps -nobreakpad

Parent is shutting down, bye...
```
How to fix this or otherwise solve this problem.

**Comment by reinerh** [comment, marked as answer]
You need to tighten the profile more (disallow stuff like dbus) and probably use private home directories (as they might store some state in ~/.steam).

**Reply by rusty-snake** [reply]
@c3zs4r that's nothing that is official supported by firejail it is a side-effect of the isolation from the sandbox, so that the sandboxed program does not know/see other instances. What @reinerh says is that this required more isolation then the default steam.profile (e.g. `dbus-user none`, `dbus-system none`, `private ${HOME}/.steam-session1`, ...).
